content/blog/eclipse-temurin-slsa2/index.md

```diff
@@ -0,0 +1,116 @@
+  used by the build process to produce the builds, the output from the
+  openjdk `configure` invocation and various other pieces of information. 
+  We are continually evolving the specific details which we include in the
+  SBOM.  If you want to join in the discussion on the content, you can find
+  the conversation in
+  [temurin-build#3013](https://github.com/adoptium/temurin-build/issues/3013)
+  or talk to us in the #secure-dev slack channel in the Adoptium workspace.
+
+## SLSA Level 2
+
+Level 2 adds in additional requirements to provide some tamper resistance of
+the build process, mostly in the areas of having all of our code version
+controlled which we do by storing anything related to producing our buids in
+git in our case and also by using a build service to produce and distribute
+those builds - in our case this is our Jenkins CI server. In our release
+cycles since July 2022 we have also provided PGP signature verification
+information along with the SHA checksums that we previously had, which was
+the final part of achieving SLSA level 2 compliance. The requirements here
+are:
+
+- **Source - Version controlled**. All of our source code is stored in GitHub in
+  the jdkXX repositories such as
+  [jdk17u](https://github.com/adoptium/jdk17u/), which we mirror from the
+  [openjdk project](https://github.com/openjdk). These
+  are version controlled and have tags for each release which we build from.
+  Contributors agree to the rules of the
+  [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)
+
+- **Build - Build service**. All of the build steps are run using our Jenkins
+  build service - currently at https://ci.adoptopenjdk.net which is used to
+  perform the builds, generate the SBOMs, and build the installers where
+  applicable. The output from the builds are then posted into GitHub release
+  repositories named as temurinXX-binaries (e.g.
+  [temurin17-binaries](https://github.com/adoptium/temurin17-binaries) and
+  also exposed via our API and web site.
+
+- **Provenance - Authenticated**. We provide GPG checksums that users can use to
+  verify that the downloads they receive have come from the Adoptium CI
+  system. The verification process is defined in
+  [this blog post](https://blog.adoptium.net/2022/07/gpg-signed-releases)
+  and most of the process in there is applicable to the SBOMs for the latest
+  releases.
+
+- **Provenance - service generated**. The GPG signatures for the binary
+  are generated as an integral part of the build process for the binaries
+  and will be done in the same way for the signing of the SBOMs - the current
+  SBOM signing was done retrospectively in our Jenkins instance. The Jenkins
+  instance hosts the private keys from the Eclipse Foundation which are used
+  to performing the signature generation.
+
+# Next steps for the project
+
+Are we finished? Absolutely not! And we are continuing to work towards
```

Comment:
`s/And we are continuing/We are continuing` reads better with the `and` that follows later on in the sentence.